src/backend/EdFi.DataManagementService.Backend.Postgresql.Test.Integration/DatabaseTest.cs

```diff
@@ -353,8 +351,14 @@ Func<NpgsqlConnection, NpgsqlTransaction, Task<U>> dbOperation2
         Transaction1Go?.WaitOne();
 
         // Step #11: Cleanup and return
-        transaction2?.Dispose();
-        connection2?.Dispose();
+        if (transaction2 is not null)
+        {
+            await transaction2.DisposeAsync();
```

Comment:
awaiting with `transaction1?.DisposeAsync()` does not compile when there is a `ValueTask` as the return instead of a `Task`. I don't know why (in detail), but the solution is simple enough: add an explicit guard clause.